pkg/kratos/handlers.go

```diff
@@ -560,12 +567,29 @@ func (a *API) handleCreateSettingsFlow(w http.ResponseWriter, r *http.Request) {
 	w.Write(resp)
 }
 
-func NewAPI(service ServiceInterface, mfaEnabled bool, baseURL string, logger logging.LoggerInterface) *API {
+func (a *API) deleteKratosSession(w http.ResponseWriter) {
+	// To delete the session we delete the kratos session cookie.
+	// This is hacky as it does not call the Kratos API and is likely to break on
+	// a new Kratos version, but there is no easy way to delete the session
+	// from the Kratos API
+	c := &http.Cookie{
+		Name:     a.kratosSessionCookieName,
```

Comment:
```
const KRATOS_COOKIE_NAME = "whatever"
```

picking up on on the point above

```suggestion
		Name:     KRATOS_COOKIE_NAME,
```